styles.css

```diff
@@ -336,3 +334,101 @@ body {
 .skills h4 {
   font-size: 20px;
 }
+
+/* Contact Form */
+
+.contact-forms {
+  font-family: Poppins, sans-serif;
+  background-image: url("images/Contact-me.svg");
+  background-repeat: no-repeat;
+  min-height: 100vh;
+}
+
+.contact {
+  margin-top: 103px;
+  text-align: center;
+}
+
+.form-heading {
+  color: #fff;
+  font-style: bold;
+}
+
+.form-paragraph {
+  margin-top: 12px;
+  margin-left: 24px;
+  margin-right: 24px;
+  text-align: center;
+  font-size: 20px;
+  color: #fff;
+}
+
+.name {
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 87px;
+  width: 327px;
+  height: 48px;
+  padding-left: 16px;
+  padding-right: 38px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  background-color: #fff;
+  border-radius: 8px;
+}
+
+.email {
+  margin-top: 20px;
+  margin-left: 24px;
+  width: 327px;
+  height: 48px;
+  padding-left: 16px;
+  padding-right: 38px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  background-color: #fff;
+  border-radius: 8px;
+  border: none;
+}
+
+.text-field {
+  margin-left: 24px;
+  margin-top: 18px;
+  width: 327px;
+  background-color: #fff;
+  border-radius: 8px;
+  padding-left: 16px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.form-btn {
+  background-color: #fff;
+  width: 140px;
+  height: 48px;
+  margin-left: 24px;
+  margin-top: 18px;
+  margin-bottom: 45px;
+  padding-top: 12px;
+  padding-right: 16px;
+  padding-bottom: 12px;
+  padding-left: 16px;
+  border-radius: 8px;
+  color: #6070ff;
+  border: none;
+  font-size: 17px;
+}
```

Comment:
- [ ] Kindly add `hover` and `active` (pressed) button interactions for the form submission button because it's a requirement.

![image](https://user-images.githubusercontent.com/68796232/161135068-30a3b994-9836-4119-8375-72499fd818bb.png)

![image](https://user-images.githubusercontent.com/68796232/161135167-9e5d9b29-8433-4ed2-a2eb-7b1d3bd09fa3.png)